discovery-infra/update_assisted_service_cm.py

```diff
@@ -36,6 +36,7 @@
     ("HARDWARE_INFO_IMAGE", ""),
     ("INVENTORY_IMAGE", ""),
     ("OCM_BASE_URL", ""),
+    ("OPENSHIFT_INSTALL_RELEASE_IMAGE", "")
```

Comment:
@YuviGold If you set this one as it is, the ConfigMap template (source file) will empty the REPLACE_XXXXX string, then the replacement will not happen on the Assisted-Service repository